src/main/resources/application-localdev.properties

```diff
@@ -0,0 +1,2 @@
+spring.kafka.ssl.trust-store-location=${TRUSTSTORE_LOCATION}
```

Comment:
Is this required since it's already in the application.properties?

Reply:
I removed it from the main application.properties file as this is only needed for localdev.